lib/cmd/fh3/services/list.js

```diff
@@ -0,0 +1,28 @@
+/* globals i18n */
+var common = require("../../../common");
+
+module.exports = {
+  'desc' : i18n._('List services'),
+  'examples' :
+  [{
+    cmd : 'fhc services list',
+    desc : i18n._('List all services')
+  }],
+  'demand' : [],
+  'alias' : {
+    'json' : 'j',
+    0 : 'json'
+  },
+  'describe' : {
+    'json' : i18n._('Output into json format')
+  },
+  'customCmd' : function(params,cb) {
+    common.listServices(function(err, services) {
+      if (err) {
+        return cb(err);
+      }
+      params._table = common.createTableForProjects(services);
```

Comment:
It is the same of the above too. The default behavior is to return the table before the changes. Following the local test without any change. 

<img width="661" alt="screen shot 2017-05-30 at 8 55 03 am" src="https://cloud.githubusercontent.com/assets/7708031/26582026/b9c6cec2-4515-11e7-801c-4aaf93982498.png">